lib/promise.js

```diff
@@ -241,6 +241,8 @@ var Promise = define(null, {
          * @return {comb.Promise} this promise for chaining
          */
         then: function (callback, errback) {
+            console.warn('Registering a `.then` is not best practice. Did you mean to use `.chain`?')
+
```

Comment:
We could `.trace` so we know where its coming from.  But this is probably good enough.

Reply:
i like that better actually.